index.js

```diff
@@ -181,7 +181,7 @@ module.exports = CachingWriter.extend({
 
     walkSync(inDir)
       .forEach(function(relativePath) {
-        if (relativePath.slice(-1) !== '/') {
+        if (relativePath.slice(-1) !== '/' && relativePath.slice(-3) === '.js') {
```

Comment:
I believe this can use only the .js condition (it is not possible to end with '/' **and** '.js').